packages/calcite-components/src/components/tabs/tabs.e2e.ts

```diff
@@ -346,4 +342,19 @@ describe("calcite-tabs", () => {
 
     expect(selectedTitleOnEmit).toBe("boats");
   });
+
+  it("inheritable props `position` and `scale` get passed to `tab-nav` and `tab-titles`", async () => {
```

Comment:
This test seems to be a partial duplicate of `should render itself and child tab elements with corresponding scale`.